### PR TITLE
Feature/78078 scope overview improve responsivenes tmp

### PR DESCRIPTION
**Pull request by Cnordbo**
Initial responsivenes improvement for Scopeoverview. 

* Scrollable filter
* Scrollable table body
* Sticky table header
* Fixed overflows

Additional: 
* Switched H1 tags to use EDS Typography component 
* Import statements sorted in touched files.

### CODE REVIEW

**Comment by bratli** [comment]
image

If I resize my window, the table is cropped.

**Comment by bratli** [comment]
Scrollable filter, kan vi få scrollbar helt inntil siden?
image

File: src/modules/Preservation/index.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { Route, BrowserRouter as Router, Switch, useRouteMatch } from 'react-router-dom';
 
 import AddScope from './views/AddScope/AddScope';
-import EditTagProperties from './views/EditTagProperties/EditTagProperties';
 import { Container } from './style';
```
Comment (bratli) [inline src/modules/Preservation/index.tsx:4]: remove

File: src/modules/Preservation/style.ts [hunk 2]
```diff
@@ -1,13 +1,7 @@
 import styled from 'styled-components';
 
 export const Container = styled.div`
```
Comment (bratli) [inline src/modules/Preservation/style.ts:3]: remove. Not in use

**Review by bratli — CHANGES_REQUESTED** [review]
See previous comments with screenshots

**Comment by arildbjerkenes** [comment]
Should the scroll-bar next to the table be thicker? At least, the scrollbars should have the same width.

**Review by arildbjerkenes — CHANGES_REQUESTED** [review]
I approve all changes, except that I think we should verify the width on the scroll bars.

**Comment by Cnordbo** [comment]
> I approve all changes, except that I think we should verify the width on the scroll bars.

No changes has been done on the width of scrollbars.

**Pull request merged**